src/index.html

```diff
@@ -5,9 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet"
+      href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <img src="./images/imac.jpeg"
+        alt="iMac"
+        class="image">
+
+        <div class="info">
+          <h1 class="info__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+          <div class="info__prod-code">Product code: 195434</div>
+
+          <div class="info__rate star">
+            <div class="rate star__4">
+              <div class="rate__star"></div>
```

Comment:
Why do you need `rate` class? It should be class `stars`
```suggestion
            <div class="stars stars--4">
              <div class="stars__star"></div>
```
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/a000664c-0c1b-4654-8818-4f487c33774b)